README_DEVELOPERS.md

```diff
@@ -6,6 +6,14 @@ Building with libgit2 is tricky atm. Run `./dmake make -C services/cd-service bi
 Afterwards run `make release`. This will push the docker image, package the helm chart and create a git tag. The helm chart must be uploaded manually to the github release at the moment.
 Afterwards bump the version in the `version` file.
 
+## Start Kuberpult with Docker Compose
+* Kuberpult can be started with `docker-compose up`
+* This only starts the services locally.
+* This **doesn't** create the repository or any data for Kuberpult.
+* This builds and runs the current code on your local machine.
+* Only requirements are `docker` and `docker-compose`
```

Comment:
With Podman and podman-compose it also works, but small modifications are necessary:

```suggestion
* Only requirements are `docker` and `docker-compose`
  - If you use Podman (and `podman-compose`) instead (e.g. on macOS), you might need small changes to the `docker-compose.yml` file:
    ```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index 2b72234..851a41d 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -5,6 +5,7 @@ services:
     ports:
       - "8080:8080"
       - "8443:8443"
+    user: 0
     volumes:
       - .:/kp/kuberpult
   frontend:
@@ -16,6 +17,7 @@ services:
       - "8081:8081"
     depends_on:
       - backend
+    user: 0
     volumes:
       - .:/kp/kuberpult
   ui:
@@ -25,5 +27,6 @@ services:
       - "3000:3000"
     depends_on:
       - frontend
+    user: 0
     volumes:
       - .:/kp/kuberpult
```
```

Using UID 0 should be fine with Podman, as it (unlike Docker) runs the container with the privileges of the current user (the UID of the current user is mapped to UID 0 inside the container).

Reply:
This suggestion actually does not work like that. If you want to include it, maybe rewrite it to say that each container needs `user: 0` for this to work in Podman (because otherwise the process in the container does not have access to the filesystem mounted from the user's home directory into the container). Maybe also add that explanation about permissions, if you add the comment at all.

Reply:
I added the comment with some explanations in the `outside of docker` section after merging with main.